docker/docker.lua

```diff
@@ -43,133 +43,136 @@ end
-					  "base_url" : 'http://127.0.0.1:65000',
-					  "version" : '0.1.0',
-					  "expected_status" : [
-					     200,
-					     201,
-					     204
-					  ],
-					  "formats" : ["json"],
-					  "methods" : {
-					     "containers" : { 
-						"path" : "/containers/json",
-						"method" : "GET",
-						"optional_params" : [
-						   'all',
-						   'limit',
-						   'since',
-						   'before',
-						   'size'
-						]
-					     },
-					     "create_container" : { 
-						"path" : "/containers/create",
-						"method" : "POST",
-						"optional_params" : [
-						   'name'],
-						"required_payload" : true,
-					     },
-					     "inspect_container" : { 
-						"path" : "/containers/:id/json",
-						"method" : "GET",
-						"required_params" : [
-						   "id"
-						],
-					     },
-					     "fs_changes_container" : { 
-						"path" : "/containers/:id/changes",
-						"method" : "GET",
-						"required_params" : [
-						   "id"
-						],
-					     },
-					     "start_container" : { 
-						"path" : "/containers/:id/start",
-						"method" : "POST",
-						"required_params" : [
-						   "id"
-						],
-					     },
-					     "stop_container" : { 
-						"path" : "/containers/:id/stop",
-						"method" : "POST",
-						"required_params" : [
-						   "id"
-						],
-					     },
-					     "restart_container" : { 
-						"path" : "/containers/:id/restart",
-						"method" : "POST",
-						"required_params" : [
-						   "id"
-						],
-					     },
-					     "kill" : { 
-						"path" : "/containers/:id/kill",
-						"method" : "POST",
-						"required_params" : [
-						   "id"
-						],
-					     },
-					     "remove_container" : { 
-						"path" : "/containers/:id",
-						"method" : "DELETE",
-						"required_params" : [
-						   "id"
-						],
-					     },
-					     "top" : { 
-						"path" : "/containers/:id/top",
-						"method" : "GET",
-						"required_params" : [
-						   "id"
-						],
-						"optional_params" : [
-						   'ps_args',
-						   ]
-					     },
-					     "images" : { 
-						"path" : "/images/json",
-						"method" : "GET",
-						"optional_params" : [
-						   'all',
-						   'limit',
-						   'since',
-						   'before',
-						   'size']
-					     },
-					     "inspect_image" : { 
-						"path" : "/images/:name/json",
-						"method" : "GET",
-						"required_params" : [
-						   "name"
-						],
-					     },
-					     "history_image" : { 
-						"path" : "/images/:name/history",
-						"method" : "GET",
-						"required_params" : [
-						   "name"
-						],
-					     },
-					     "info" : { 
-						"path" : "/info",
-						"method" : "GET",
-					     },
-					     "version" : { 
-						"path" : "/info",
-						"method" : "GET",
-					     },
-					     "search_images" : { 
-						"path" : "/images/search",
-						"method" : "GET",
-						"required_params" : [
-						   "term"
-						],
-					     },
-					  }
-				       }]]
+	"base_url" : 'http://127.0.0.1:65000',
```

Comment:
The spacing should be the only fix - I think that was just my editor making things readable.